assets/pages/monthly-contributions-existing/monthlyContributionsExisting.scss

```diff
@@ -0,0 +1,68 @@
+#monthly-contributions-existing-page {
+
+	a {
+		color: gu-colour(neutral-1);
+	}
+
+	// ----- Headings
+
+	.existing {
+		background-color: darken(gu-colour(multimedia-main-2), 5%);
+	}
+
+	.existing__content {
+		background-color: gu-colour(multimedia-main-2);
+		padding: $gu-v-spacing $gu-h-spacing;
+	}
+
+	.existing__wrapper {
+
+		@include mq($from: desktop) {
+			margin-left: 20%;
+			margin-right: 20%;
+		}
+	}
+
+	.existing__heading {
+		font-family: $gu-egyptian-web;
+		color: gu-colour(neutral-1);
+		font-size: 44px;
+		line-height: 46px;
+		font-weight: bold;
+	}
+
+	.existing__subheading {
+		font-family: $gu-egyptian-web;
+		font-size: 40px;
+		line-height: 42px;
+		font-weight: lighter;
+		color: gu-colour(neutral-1);
+	}
+
+	.component-cta-link {
+		color: gu-colour(neutral-1);
+		background: transparent;
+		display: inline-block;
+		width: 180px;
+		border: 1px solid gu-colour(neutral-1);
+		font-weight: normal;
+		margin-bottom: $gu-v-spacing * 4;
+		margin-top: 4 * $gu-v-spacing;
+
+		svg {
+			fill: gu-colour(neutral-1);
+			// Fix for IE10.
+			left: 190px;
```

Comment:
By apply do you mean does it only get used on IE10? No, it's applied on all browsers, but only IE10 needs it.